### PR TITLE
Update the java project document

### LINKED ISSUE #4948: Outdated GIF for "Add a JAR"
On the ["Add a JAR" section of the Java projects documentation](https://github.com/microsoft/vscode-docs/blob/24e218fee4d41d8e19bd1edf9c443b4254fda145/docs/java/java-project.md#add-a-jar), the GIF shown is using an outdated version of the extension where the Java Projects view is still called "Java Dependencies". This is a minor issue, but could be confusing for new users (and all the other GIFs/screenshots are updated).

### PR DESCRIPTION
**Pull request by jdneo**
For the JDK part, I will update it in another PR.

This PR also removes some out-of-date images and videos, which can fix #4948

Signed-off-by: Sheng Chen <sheche@microsoft.com>

### CODE REVIEW

**Comment by jdneo** [comment]
@nickzhums please review

**Review by minhhieuuuuu88 — APPROVED** [review]

File: docs/java/java-project.md [hunk 1]
```diff
@@ -9,49 +9,43 @@ MetaDescription: Lightweight Mode, Maven Support, Java Package and Dependency Ma
 MetaSocialImage:
 ---
 
-# Java project management in VS Code
+# Manage Java Projects in VS Code
```
Comment (nickzhums) [inline docs/java/java-project.md:12]:
```suggestion
# Managing Java Projects in VS Code
```

File: docs/java/java-project.md [hunk 2]
```diff
@@ -9,49 +9,43 @@ MetaDescription: Lightweight Mode, Maven Support, Java Package and Dependency Ma
...
 
 <a class="tutorial-install-extension-btn" href="vscode:extension/vscjava.vscode-java-pack">Install the Extension Pack for Java</a>
 
-For details on how to get started with the extension pack, please see [Getting Started](/docs/java/java-tutorial.md).
+> For details on how to get started with the extension pack, please see [Getting Started](/docs/java/java-tutorial.md).
 
 ### Project view
 
-Project view helps you view your project and its dependencies, and provides entry points for project management tasks. You can switch between a hierarchy view and flat view.
+The project view helps you view your Java projects and their dependencies, and provides entry points for project management tasks.
```
Comment (nickzhums) [inline docs/java/java-project.md:22]: Should we add an intro to where project view is? something like this:

The Java project view is displayed in the explorer view and you can toggle its display by clicking "..."  on the explorer area and selecting "Java Project"

File: docs/java/java-project.md [hunk 3]
```diff
@@ -9,49 +9,43 @@ MetaDescription: Lightweight Mode, Maven Support, Java Package and Dependency Ma
...
+![inline button](images/java-project/inline-button.png)
 
-A project or module is imported to a workspace through **File** > **Open Folder** or **File** > **Open Workspace** menu. VS Code for Java will detect your project type automatically. As a tip, you can run the command **Java: Import Java projects in workspace** to reimport a project and alert the language server that there were changes to the project, without reloading your window.
+There are also many useful features in each node's context menu, you can right click on the node in explorer to toggle it out.
 
-### Add a dependency
+![context menu](images/java-project/context-menu.png)
 
-For Maven project, you can add a dependency by clicking the **+** sign next to **Maven Dependencies** node in project view.
+### Create Java project
```
Comment (nickzhums) [inline docs/java/java-project.md:38]:
```suggestion
### Create a new Java project
```

File: docs/java/java-project.md [hunk 4]
```diff
@@ -9,49 +9,43 @@ MetaDescription: Lightweight Mode, Maven Support, Java Package and Dependency Ma
...
+You can create a new Java project by clicking the **+** button in the navigation bar, or through the command: **Java: Create Java Project...** in Command Palette (`kb(workbench.action.showCommands)`). During creation, VS Code will facilitate installing required extension(s) per your project type, if the extension(s) weren't installed.
 
-### Add a JAR
+![create project](images/java-project/projectmanager-createproject.png)
 
-JAR file(s) can be added by clicking the **+** sign next to **Referenced Libraries** node in project view.
+### Import Java projects
 
-![Add Dependency](images/java-project/manage-dependencies.gif)
+Projects will be imported to workspace through **File** > **Open Folder...** (Make sure the opened folder contains your build tool scripts, i.e. `pom.xml` or `build.gradle`). VS Code for Java will detect your projects and import them automatically.
```
Comment (nickzhums) [inline docs/java/java-project.md:46]:
```suggestion
You can directly import existing Java projects to workspace through **File** > **Open Folder...** (Make sure the opened folder contains your build tool scripts, i.e. `pom.xml` or `build.gradle`). VS Code for Java will detect your projects and import them automatically.
```

File: docs/java/java-project.md [hunk 5]
```diff
@@ -9,49 +9,43 @@ MetaDescription: Lightweight Mode, Maven Support, Java Package and Dependency Ma
...
 
-For more details on library, refer to [Library Configuration](/docs/java/java-project.md#library-configuration).
+When you add a new module into your projects, you can trigger the command **Java: Import Java projects in workspace** to import them to workspace.
```
Comment (nickzhums) [inline docs/java/java-project.md:48]: we should probably say both project and modules can be imported using "Open Folder"

Comment (nickzhums) [inline docs/java/java-project.md:48]: should we also mention the effect of "import project in workspace"? the purpose is to notify language server without window reload

File: docs/java/java-project.md [hunk 6]
```diff
@@ -109,7 +103,28 @@ If you only work with source files and don't use a build tool, VS Code will appl
...
 ![jdk home](images/java-project/projectmanager-javahomeconf.png)
 
+## Configure classpath for unmanaged folder
+We provide a user interface to configure the classpath for unmanaged folder. The classpath can be set manually in the classpath configuration page. You can open it by executing the **Java: Configure Classpath** command from the Command Palette (`kb(workbench.action.showCommands)`).
+
+![configure classpath](images/java-project/configure-classpath.png)
+
+In some rare cases, you may need to clean the Java workspace by executing the **Java: Clean Java Language Server Workspace** command from the Command Palette (`kb(workbench.action.showCommands)`) to let the language server rebuild your dependencies.
+
 ## Library configuration
```
Comment (nickzhums) [inline docs/java/java-project.md:113]: Dependency Management better term than Library configuration?

File: docs/java/java-project.md [hunk 7]
```diff
@@ -173,14 +188,6 @@ If you want to manually specify a JAR file as a source attachment, you can provi
...
-
-```xml
-<classpathentry kind="lib" path="lib/log4j.jar"/>
-```
-
-In some rare cases, you may need to clean the Java workspace by executing the **Java: Clean Java Language Server Workspace** command from the Command Palette (`kb(workbench.action.showCommands)`) to let the language server rebuild your dependencies.
-
 ## Lightweight Mode
 
 VS Code for Java supports two modes, lightweight and standard. With lightweight mode, only source files and JDK are resolved by the language server; with standard mode, imported dependencies are resolved and the project is built by the language server. Lightweight mode works best when you need a super quick-to-start and lightweight environment to work with your source files, for example, reading source code, navigating among source code and JDK, viewing outline and Javadoc, and detecting and fixing syntax errors. Also, code completion is supported within the scope of source files and JDK.
```
Comment (nickzhums) [inline docs/java/java-project.md:193]:
```suggestion
VS Code for Java supports two modes, lightweight and standard. With lightweight mode, only source files and JDK are resolved by the language server; with standard mode, imported dependencies are resolved and the project is built by the language server. Lightweight mode works best when you need a quick-start and lightweight environment to work with your source files, for example, reading source code, navigating among source code and JDK, viewing outline and Javadoc, and detecting and fixing syntax errors. Also, code completion is supported within the scope of source files and JDK.
```

File: docs/java/java-project.md (hunk 2 above)
Comment (jdneo) [reply docs/java/java-project.md:22]: How about:

By default, the Java project view is displayed below the explorer view. If you cannot see it. Try clicking the `...` button next to the `EXPLORER` and select `Java Projects`.

File: docs/java/java-project.md (hunk 5 above)
Comment (jdneo) [reply docs/java/java-project.md:48]: ok

File: docs/java/java-project.md (hunk 2 above)
Comment (nickzhums) [reply docs/java/java-project.md:22]: sounds good!

**Review by nickzhums — APPROVED** [review]

**Pull request merged**